[master] fix: shrinking input field for oci runtimes

klocke-io opened the pull request:
**What this PR does / why we need it**:
Improves usability of the select dialog in the worker group for `Additional OCI Runtimes`
**Which issue(s) this PR fixes**:
Fixes # https://github.com/gardener/dashboard/issues/2426

**Special notes for your reviewer**:
The issue is reproduced in : 
[Vuetify Playground](https://play.vuetifyjs.com/#eNq9ld9r2zAQx/+Vq1+8QeSkCYUucwclsB8PYzAGfWj6oNgXR0yWhCQnDSH/+052nSWO08Fgw2Ck00e68935q8dd5Gw2vDcmWVcYTaPUY2kk9/hhrgDSNePG1MN6kmnluVBoX0xkzMUaMsmdu5tHOVtKfJ5Hh9XedQgvtrHcnJAd1mJRSW6ZUKbyHZDQK8ZAW1EIxeUUgLE+4KJzqzd0ZM+uVzeA81uJtFIKxTYi96spjG9G5vSL+wOk5DmUmHnYCL/SlW/cnIdwhHYX4BBaHVNBQbFRE59bWaF+stE8Ot+0ZqXOUYa4XXEBQOd4gY4KLLUlcgf1aArxhlslVBHDvm9ryxsrSm63fYigjjqc+wo3DWD4uMeYxwOIF+GVxU99rOSL+oNmbT/C90p5UWIfvBLKE7tAcOTbQx9j0DrhPCrPAt6TI24Fr8+pVI5WktP8/KDhf+6DUrLrPzfAUWZnXIqlporWKZ6FqvBch/FHGoq8Nn9CbYuG+IHP3IXBw1ZT1xf95chWwrhzc1lJL4zEf9o2bSvc57nwQlOW4dvsS9sPrjdcqR1fSGQX4v67WqdDEo9TPetYTufHs3TY1dZgaaQ3HR5JMk1dZoXx4NBXjTKL0mhq6x1YXMIellaXEJOaxwEPuVbOA/36cBeIN/FnlFLDg7Yyv4rfHiHcfA1C8cJ1fkMC02Hjm7xGg/reKCngJHOOLo7kRLBhF84tOel8o5OTEenk+9r4Wzyvb1tj6GGa08Mrr2uT4VRRVZB1XEN78rofRJPkJrm+Jf+T5F0yjgZLLh020bS3WGN7+gWgYezD)

**Release note**:
<!--  Write your release note:
1. Enter your release note in the below block.
2. If no release note is required, just write "NONE" within the block.

Format of block header: <category> <target_group>
Possible values:
- category:       breaking|feature|bugfix|doc|other
- target_group:   user|operator|developer|dependency
-->
```bugfix user

```

klocke-io committed:
fix: shrinking input field for oci runtimes

grolu commented:
I just tested it. Unfortunately it does not behave correctly in case there is no container runtime value set or if the selected container runtime does not have additional runtimes. You try this by removing the cri section from the shoot spec or replacing container runtime with an arbitrary name like `fooruntime` in the create shoot yaml editor

klocke-io committed:
fix: shrinking input field for oci runtimes

klocke-io committed:
fix: shrinking input field for oci runtimes

klocke-io commented:
@grolu thanks for your feedback :) 

I addressed those concerns in ccfc7223b159fe28bd3058f0772e8e1db639b328 & 5e72547cc5ed5882831f3a9c35e3ca826b456b47

petersutter commented on frontend/src/components/ShootWorkers/GContainerRuntime.vue:
```diff
@@ -24,10 +27,11 @@ SPDX-License-Identifier: Apache-2.0
       color="primary"
       item-color="primary"
       :items="criContainerRuntimeTypes"
-      label="Additional OCI Runtimes"
```
is it possible then to have a tooltip for the title? 🤔

grolu replied:
Input field for the title is not possible as far as I know. But yes we could add it to the component (please use v-tooltip directive)

klocke-io replied:
@petersutter I see you point, I personally would find `Add. OCI Runtimes` descriptive enough. And would maybe also find it a bit unintutive for the user that this one element in the worker dialog now has a tooltip. But I have no strong feelings for this either. If feel it would help, I happily add it.

petersutter reviewed (approved):
/lgtm

petersutter commented:
please rebase from master to resolve the pending github action check

petersutter commented on frontend/src/components/ShootWorkers/GContainerRuntime.vue:
```diff
@@ -5,10 +5,13 @@ SPDX-License-Identifier: Apache-2.0
     <v-select
       v-model="criName"
       v-messages-color="{ color: 'warning' }"
+      style="flex: 0 0 125px"
```
```suggestion
```

klocke-io committed:
fix: drop style for first element to facilitate scaling when the second element is not displayed

Co-authored-by: Peter Sutter <peter.sutter@sap.com>

petersutter commented:
/cherry-pick hotfix-1.81
